ConnectorActionableMsgs/02 Cards and Actions/Demos/03-Expense-Approval/readme.md

```diff
@@ -0,0 +1,319 @@
+# Cards and Actions Using Outlook Actionable Messages – 300 Level
+----------------
+In this lab, you will walk through building an Actionable Message card and adding actions. 
+
+## Prerequisites
+
+This lab uses Visual Studio 2017. It also requires an Office 365 subscription with an active mailbox and a **Microsoft Azure** subscription.
+
+## Setup
+
+This lab will use an Azure Web App to deploy an application. The URL of the web app is required. Visit the [Microsoft Azure Portal](https://portal.azure.com) and create a new Web App. Copy the URL (for example, https://mywebapp.azurewebsites.net) for later use.
```

Comment:
It might be a good idea to refer back to the 01 module, where they were instructed to create a web app. Remind them that they can reuse that one :)